api/schemas/CANPredictionTimeseriesRow.json

```diff
@@ -37,6 +37,16 @@
       "title": "Cumulativeinfected",
       "description": "Number of cumulative infections",
       "type": "integer"
+    },
+    "cumulativePosTested": {
```

Comment:
nit: "Tested" => "Tests" maybe? since:
1. I think we need a noun in here. :-) I guess "Tested" _can_ be a noun, but 1) it's ambiguous, and 2) Since people can be tested multiple times, it's probably clearer to use "Tests"
2. "positive tested" and "negative tested" are just unusual phrasing.

Also, I'm rarely a fan of abbreviations unless they're ubiquitous.  So maybe go ahead and do `cumulativePositiveTests` and `cumulativeNegativeTests`?  Our JSON files will always be compressed (gzip) on the wire, so we don't need to work too hard to save bytes. :-)